DQM/Physics/src/QcdPhotonsDQM.cc

```diff
@@ -340,7 +340,6 @@ void QcdPhotonsDQM::analyze(const Event& iEvent, const EventSetup& iSetup) {
   float photon_eta = -9.0;
   float photon_phi = -9.0;
   bool photon_passPhotonID = false;
-  bool found_lead_pho = false;
```

Comment:
There are exactly three uses of this variable: declaration here, check on [line 392](https://github.com/cms-sw/cmssw/pull/46247/files#diff-e94d5d27fa83b3d972dc7ededb64433cefde9e45909daa42120b8f314a0d2b42L392) and setting to true on (line 393)[https://github.com/cms-sw/cmssw/pull/46247/files#diff-e94d5d27fa83b3d972dc7ededb64433cefde9e45909daa42120b8f314a0d2b42L393]